install/installConfig.php

```diff
@@ -1258,10 +1192,58 @@ private function getFormItems(
 </table>
 </div>
 EOQ;
+        $out .= "</div>";
+
+        // --------------------------
+        //  Advanced Database Configuration
+        // --------------------------------->
 
-        $out.= "</div>";
+        require_once('install/suite_install/collations.php');
 
+        $collationCB = "<select name='setup_db_collation' id='setup_db_collation' class='select' onChange='document.getElementById(\"setup_db_charset\").value = document.getElementById(\"setup_db_collation\").value.split(\"_\")[0];'>";
+        $charset = "<select name='setup_db_charset' id='setup_db_charset' class='select'>";
+
+        if (isset($collations)) {
+            if (isset($_SESSION['setup_db_type']) && $_SESSION['setup_db_type'] == "mysql") {
```

Comment:
This statement can be merged with the parent. I would also check to see if we can use strict comparison here.